### ISSUE TITLE
Documentation on how to set up spell checking

### ISSUE THREAD

**Issue by Dietr1ch** [cachix/git-hooks.nix#520]
It seems that hunspell is not that easy to get working (#159 sort of complains about it)

I can't get `hunspell` to work either, it simply skips all files, so I'm guessing that it might be good to have an example on how to use hunspell too.

```nix
# https://devenv.sh/reference/options/#pre-commit
pre-commit = {
  hooks = {
    # Spelling
    hunspell = {
      enable = true;
    };
  };
};
```

After some fooling around I ended up with,
```nix
env = {
  DICTIONARY = "en_GB";
};

pre-commit = {
  hooks = {
    # Spelling
    hunspell = {
      enable = true;
      description = "Run hunspell on all org files";
      files = "\.org$";
      entry = "${pkgs.hunspell}/bin/hunspell -p .spelling/dictionary -l";
    };
  };
};
```

That `-l` flag made me trip, as it's key and without it hunspell runs in interactive mode and results in a busy wait when running `pre-commit run --all-files --show-diff-on-failure`.

**Comment by sandydoo** [comment]
What was the original issue in your case? `.org` files [being filtered out](https://github.com/cachix/git-hooks.nix/blob/3c3e88f0f544d6bb54329832616af7eb971b6be6/modules/hooks.nix#L2794) by default or something else?

**Comment by Dietr1ch** [comment]
Yeah, org files were not checked by default, but I also wanted to specify the personal dictionary as things like `file.json` trip spellchecking.
